hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeContext.java

```diff
@@ -46,12 +46,11 @@
 import static java.util.Arrays.asList;
 import static java.util.Collections.unmodifiableList;
 
-@PrivateApi
 public class DefaultNodeContext implements NodeContext {
 
     public static final List<String> EXTENSION_PRIORITY_LIST = unmodifiableList(asList(
             "com.hazelcast.instance.EnterpriseNodeExtension",
```

Comment:
Need to change the EE class here.

Reply:
@pveentjer I think this was missed? On current `master`, `EnterpriseNodeExtension` is not created due to wrong EE class name.